torch/fx/traceback.py

```diff
@@ -0,0 +1,40 @@
+from contextlib import contextmanager
+from typing import Optional
+from ._compatibility import compatibility
+
+__all__ = ['override_stack_trace', 'current_stack_trace', 'is_stack_trace_overridden']
+
+active_interpreter = None
+is_overriden = False
+
+@compatibility(is_backward_compatible=False)
+@contextmanager
+def override_stack_trace(interpreter):
+    global active_interpreter
+    global is_overriden
+
+    saved_intepreter = active_interpreter
+    saved_is_overriden = is_overriden
+    try:
+        active_interpreter = interpreter
+        is_overriden = True
+        yield
+    finally:
+        active_interpreter = saved_intepreter
+        is_overriden = saved_is_overriden
+
+@compatibility(is_backward_compatible=False)
+def current_stack_trace() -> Optional[str]:
```

Comment:
Could we make this the exact same API as `traceback.format_stack()` ? That way, we can simply replace `traceback.format_stack()` calls with `torch.fx.format_stack()` in our code and it will just work (I'm thinking about anomaly mode in particular).

Reply:
Sure, done.